rollup.config.js

```diff
@@ -0,0 +1,52 @@
+import path from "path";
+
+import alias from "@rollup/plugin-alias";
+import babel from "@rollup/plugin-babel";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import analyze from "rollup-plugin-analyzer";
+import cleaner from "rollup-plugin-cleaner";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+import { globSync } from "glob";
+import { keys } from "ramda";
+
+import packageJSON from "./package.json";
+const rootResolve = require("./resolve.js");
+
+const globals = Object.fromEntries(
+  keys(packageJSON.peerDependencies).map(lib => [lib, lib])
+);
+
+export default {
+  input: "./src/index.js",
+  output: [
+    {
+      inlineDynamicImports: true,
+      file: `index.mjs`,
+      format: "esm",
```

Comment:
We might need another bundle with cjs format, since we are planning to use this in neetoUI and it has jest tests.